### PR TITLE
landing page

### PR DESCRIPTION
**Pull request by MariaBjork**
created button component and working on landing page

### CODE REVIEW

File: src/components/button.tsx
Comment (utlandingur) [inline src/components/button.tsx:1]: This is fine, but as a button we should instead allow children so we could also have an image, or maybe and image or text.

File: src/App.tsx
Comment (utlandingur) [inline src/App.tsx:1]: {!firstTime ? <><Button text="CONTINUE"/>
      <Button text="CHAPTERS"/>
      <Button text="START GAME IN ANOTHER LANGUAGE"/></>: <Button text="START"/>}
      
The empty react element that should wrap the whole piece.
The state is ok, but where are we grabbing the firstTime from? LocalStorage or a database?
we can use a library i use for localState if thats what we want
https://www.npmjs.com/package/react-session-hooks

**Review by utlandingur — DISMISSED** [review]

File: src/App.tsx
Comment (utlandingur) [reply src/App.tsx:1]: Why are we passing true to textToSpeech? It requires a string

File: src/pages/LandingPage/LandingPage.tsx
Comment (utlandingur) [inline src/pages/LandingPage/LandingPage.tsx:1]: We're not using any state (and firstTime is always true in this) so we can just remove state

**Review by Evelina-Ok — APPROVED** [review]

**Pull request merged**